Standards/CCSMB-5.md

```diff
@@ -0,0 +1,23 @@
+# CCSMB-5: Radio Transmissions
+*Author: Emma Beurskens <@emmaknijn>*
+*Version: v1.0.0*
+*Last updated: YYYY-MM-DD*
+
+## Design
+A client will only listen to packets while the audio is being transmitted from a server
+The server will communicate to the client over modem
+
+## Packet
+A packet consists of the following fields in a table:
+- buffer: A table that contains Signed 8-bit PCM @ 48kHz audio
+- station: A string that contains the name of the radio station
+- title: A string that contains the song that is currently being played
```

Comment:
Is this required for all data? What happens if there's no "song" playing? Passing a blank string feels like a dirty solution.

Reply:
The metadata table system would resolve this